ev3dev/sensor/lego.py

```diff
@@ -430,6 +430,18 @@ def reset(self):
         self.mode = self.MODE_GYRO_ANG
         self._direct = self.set_attr_raw(self._direct, 'direct', 17)
 
+    def wait_until_angle_changed_by(self, delta):
+        """
+        Wait until angle has changed by specified amount.
+        """
+        mode = self.mode
+        assert mode == self.MODE_GYRO_G_A or mode == self.MODE_GYRO_ANG or\
+            mode == self.MODE_TILT_ANG,\
```

Comment:
You can save a few CPU cycles via `assert mode in (self.MODE_GYRO_GA, self.MODE_GYRO_ANG, self.MODE_TILT_ANG)`

Reply:
Thanks, I've made that change.